src/schema/v2/me/collectionsConnection.ts

```diff
@@ -18,16 +18,22 @@ export const CollectionsConnection: GraphQLFieldConfig<any, ResolverContext> = {
   args: pageable({
     page: { type: GraphQLInt },
     size: { type: GraphQLInt },
+    default: { type: GraphQLBoolean },
+    saves: { type: GraphQLBoolean },
   }),
   resolve: async (parent, args, context, _info) => {
     const { id: meID } = parent
     const { collectionsLoader } = context
     if (!collectionsLoader) return null
 
     const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+    const { default: default_, saves } = args // 'default' is a reserved word in JS
+
     const { body, headers } = await collectionsLoader({
       user_id: meID,
       private: true,
+      default: default_,
+      saves,
```

Comment:
**suggestion**: what do you think about this?

```javascript
const { body, headers } = await collectionsLoader({
  user_id: meID,
  private: true,
  default: args.default,
  saves: args.saves,
  size,
  offset,
  total_count: true,
})
```

Reply:
Yeah, that's nicer — changed in 4f34425